web-server/v0.4/src/pages/Search/SearchList.js

```diff
@@ -210,12 +200,12 @@ export default class SearchList extends Component {
                     mode="multiple"
                     style={{ width: '100%' }}
                     placeholder="Select index"
-                    value={selectorIndices}
+                    value={selectedIndices}
                     onChange={this.updateSelectedIndices}
                     tokenSeparators={[',']}
                   >
-                    {indices.map((index, i) => {
-                      return <Select.Option key={i}>{index}</Select.Option>;
+                    {indices.map((month) => {
```

Comment:
This looks kinda similar to the code in the Controllers page, any possibility of refactoring this out into a component?

Reply:
@portante Yes we should reuse the `MonthSelect` component I've defined in PR #1130 to replace this Select reference. I'll update that PR with those changes to SearchList.js.

Reply:
We may want to merge this commit first so we can rebase PR #1130 on top of the modified Redux logic before I make that change.